modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserLocalServiceTest.java

```diff
@@ -120,7 +120,7 @@ public static void tearDownClass() {
 	@Test
 	public void testAuthenticateByEmailAddress() throws Exception {
 		NoticeableFuture<User> userNoticeableFuture =
-			_noticeableExecutorService.submit(() -> UserTestUtil.addUser());
+			_noticeableExecutorService.submit(UserTestUtil::addUser);
```

Comment:
```
NoticeableFuture<User> userNoticeableFuture =
	_noticeableExecutorService.submit((Callable<User>)UserTestUtil::addUser);

```

This can pass compilation.
But SF is not smart enough to do type casting.

I don't konw how to make it do the right thing, I need time the figure it out.